index.php

```diff
@@ -1,40 +1,140 @@
 <html>
     <head>
     <title> Play </title>
-    <link rel='stylesheet' href='gamestyle.css'>
-        <script>
+    <link rel='stylesheet' href='gamestyle.css'/>      
+    </head>
+    <body style='margin:0; overflow: hidden'>
+    <div id='wrapper' class = 'wrapper'></div>
+    <script>
+        var player;
+        var playerX = 400;
+        var playerY = 200;
+    
+        function paintScreen(){
+        var wrapper = document.getElementById('wrapper');
+        wrapper.innerHTML = `
+        <div id='wrapper'>
+        <div style='background-color: white; height: 60px;'></div>`
+        var conveyorBelt = document.createElement('div');
+        conveyorBelt.className = 'conveyorBelt';
+        conveyorBelt.innerHTML =`
+        <div class='beltRolls'></div>
+        <div class='beltRolls'></div>
+        <div class='beltRolls'></div>
+        <div class='beltRolls'></div>
+        <div class='beltRolls'></div>
+        <div class='beltRolls'></div>
+        <div class='beltRolls'></div>
+        <div class='beltRolls'></div>
+        <div class='beltRolls'></div>
+        <div class='beltRolls'></div>
+        <div class='beltRolls'></div>
+        <div class='beltRolls'></div>
+        <div class='beltRolls'></div>
+        <div class='beltRolls'></div>`;
+        wrapper.appendChild(conveyorBelt);
+        var items = [];
+        items[0] = document.createElement('div');
+        items[1] = document.createElement('div');
+        items[2] = document.createElement('div');
+        items[3] = document.createElement('div');
+        items[0].className = 'item1';
+        items[1].className = 'item2';
+        items[2].className = 'item3';
+        items[3].className = 'item4';
+        for(let i=0; i<items.length; i++){
+            conveyorBelt.appendChild(items[i]);
+        }
+        var containersDiv = document.createElement('div');
+        containersDiv.className = "containers";
+        document.body.appendChild(containersDiv);
+        var containers = [];
+        containers[0] = document.createElement('div');
+        containers[1] = document.createElement('div');
+        containers[2] = document.createElement('div');
+        containers[3] = document.createElement('div');
+        containers[0].className = 'bin1';
+        containers[1].className = 'bin2';
+        containers[2].className = 'bin3';
+        containers[3].className = 'bin4';
+        for(let i=0; i<containers.length; i++){
+            containersDiv.appendChild(containers[i]);
+        }
+
+        
+
+        player = document.createElement('div');
+        player.className = 'player';
+        player.style.top = playerY + 'px';
+        player.style.left = playerX + 'px';
+        document.body.appendChild(player);
```

Comment:
I downloaded your repo and took a look - I think the reason you're seeing some weird results from the player object hitting the edge of the screen isn't because of your movement logic (I think that looks good!) it's because of what shape the browser thinks your player is.

If you go into your inspector and hover over this div, you can see what I mean, but I'll post what it looks like on my end:
<img width="212" alt="image" src="https://github.com/kchikemorah/personal_blog/assets/8910031/ff13f219-52e6-4bb9-b47d-cce7322a2949">

The fun part here is that I don't totally remember how to fix this 😅 but if you can figure out why that shape doesn't line up w/ what your player looks like, I think it'll solve the issues you're seeing.